images/github-cli/Dockerfile

```diff
@@ -7,7 +7,13 @@ RUN apt-get update -qq && apt-get install -y ca-certificates git wget
 ADD https://cli.github.com/packages/githubcli-archive-keyring.gpg /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN chmod 644 /usr/share/keyrings/githubcli-archive-keyring.gpg
 RUN echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" > /etc/apt/sources.list.d/github-cli.list
-RUN apt-get update -qq && apt-get install -y gh
+RUN apt-get update -qq && \
+    apt-get install -y \
```

Comment:
optional: consider telling apt not to install additional "recommended" packages — sometimes this can save significant space and time.
```suggestion
    apt-get install -y --no-install-recommends \
```